cmd/aries-agent-mobile/pkg/wrappers/command/aries.go

```diff
@@ -153,7 +153,7 @@ func prepareFrameworkOptions(opts *config.Options) ([]aries.Option, error) {
 	return options, nil
 }
 
-func getOutBoundTransportOpts(transport string) ([]aries.Option, error) {
+func getOutBoundTransportOpts(transport string, websocketReadLimit int64) ([]aries.Option, error) {
```

Comment:
aren't you setting default here when it is zero?

Reply:
the value is set explicitly only if it's > 0, otherwise it's delegated to `nhooyr.io/websocket` lib cc @sudeshrshetty